paymentsheet/src/test/java/com/stripe/android/lpmfoundations/paymentmethod/AddPaymentMethodRequirementTest.kt

```diff
@@ -219,8 +193,6 @@ internal class AddPaymentMethodRequirementTest {
 
     @Test
     fun testInstantDebitsReturnsFalseIfOnlyCardFundingSource() {
```

Comment:
The server-side feature flag will only send `card` as a funding source if Instant Debits is disabled, thereby acting as a de-facto feature flag.